bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -118,6 +118,8 @@ struct ReplicaConfig {
 
   // Metrics dump interval
   uint64_t metricsDumpIntervalSeconds = 600;
+
+  bool multiSignatureEnabled = false;
```

Comment:
Naming: I think this name is misleading, since it's actually controls the behavior of checking the initial key exchange